libpod/runtime_ctr.go

```diff
@@ -1259,7 +1272,25 @@ func (r *Runtime) GetContainers(loadState bool, filters ...ContainerFilter) ([]*
 		}
 	}
 
-	return ctrsFiltered, nil
+	return ctrsFiltered
+}
+
+// Apply container filters on bunch of external container lists
+func (r *Runtime) ApplyExternalContainersFilters(containersList []*contEntities.ListContainer, filters ...ExternalContainerFilter) []contEntities.ListContainer {
+	ctrsFiltered := make([]contEntities.ListContainer, 0, len(containersList))
+
+	for _, ctr := range containersList {
+		include := true
+		for _, filter := range filters {
+			include = include && filter(ctr)
+		}
+
+		if include {
+			ctrsFiltered = append(ctrsFiltered, *ctr)
+		}
+	}
+
+	return ctrsFiltered
```

Comment:
Same here, this does not depend on the libpod runtime in anyway so I don't think we should have this in libpod unless @mheon really want this here.

Reply:
Is there a reason it needs to be a Runtime method? I don't see any use of Runtime in here. If it doesn't, it should be moved.

Reply:
Yes, you're right, that's my failure I used `Runtime` receiver, I wanted to move filtering logic into separate function.

Let me keep it as separate non-exported function or do you want to put filtering logic into `Runtime.GetContainers()` function back?